notes/stories/film/film-scifi-1990s.st.txt

```diff
@@ -12041,6 +12041,105 @@ fear [There were pointed moment when Jillian was in terror of her husband, Spenc
 incorporeal being [Spencer's body had been taken over by a malevolent alien entity that, at least according to Sherman Reese, was traveling space as a transmission at the speed of light.]
 
 
+movie: Thrill Seekers (1999)
+============================
+
+:: Title
+Thrill Seekers
+
+:: Description
+Thrill Seekers (also known as The Time Shifters) is a 1999 science fiction
+television film. In the film a TV news reporter discovers that people are
+traveling back in time to watch disasters as a form of entertainment.
+
+:: Date
+1999-10-17
+
+:: References
+https://en.wikipedia.org/wiki/Thrill_Seekers_(film)
+
+:: Ratings
+2 <paul> [TV film]
+
+:: Choice Themes
+time traveler meddling in history [The premise of the film is that people are traveling back in time to watch deadly disasters for entertainment, but in so doing the future ends up getting altered in various ways.]
+
+:: Major Themes
+journalism [The TV news reported Tom Merrick uncovered incontrovertible evidence that people from the future were traveling back in time to spectate deadly disasters for entertainment.]
+violence in the media [A point was hammered home about the seemingly deranged practice of traveling back in time to watch people die horribly in historical disasters was not so different from "tuning into the six o'clock news".]
+facing organized pursuit [Tom was on the run from the FBI throughout the better part of the film.]
+the thrill of danger [The titular "thrill seekers" were people traveling back in time to watch people die horribly in disasters for kicks.]
```

Comment:
That's just sadism, not thrill of danger unless the time travelers did it for the sake of danger to themselves too....

Reply:
It's portrayed as thrill seeking in the movie - and they did put themselves in harms way in the process. I guess the comment can elaborate on the idea in more detail.

Reply:
Yeah, if they were participating in a deadly spectacle it might apply.

Reply:
Comment elaborated upon.